src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -1059,9 +1130,9 @@ public static bool ContainsTypeParameter(this TypeSymbol type, MethodSymbol para
         private static readonly Func<TypeSymbol, Symbol, bool, bool> s_isTypeParameterWithSpecificContainerPredicate =
              (type, parameterContainer, unused) => type.TypeKind == TypeKind.TypeParameter && (object)type.ContainingSymbol == (object)parameterContainer;
 
-        public static bool ContainsTypeParameters(this TypeSymbol type, HashSet<TypeParameterSymbol> parameters)
+        public static bool ContainsTypeParameters(this TypeSymbol type, HashSet<TypeParameterSymbol> parameters, bool visitCustomModifiers = false)
```

Comment:
>ContainsTypeParameters [](http://example.com/codeflow?start=27&length=22)

If we go with suggestion [https://github.com/dotnet/roslyn/pull/58288#discussion_r773990388](https://github.com/dotnet/roslyn/pull/58288#discussion_r773990388), we don't need this helper and, therefore, do not need to modify it.

Reply:
Sure, let's make the `VisitType` correct first.